README.md

```diff
@@ -42,6 +42,18 @@ Build instructions and more details can be found in the
 * Step 1: Make a space flight simulator
 * Step 2: Bloat it with features
 
+## Compiling for Windows on Visual Studio
+You will need: cmake, Visual Studio
+* Clone the repo with `https://github.com/TheOpenSpaceProgram/osp-magnum`
+* Open the repo `cd osp-magnum`
+* Get submodules `git submodule init && git submodule update`
+* Create the build folder `mkdir build && cd build`
+* Download [SDL2 visual studio development binaries](https://www.libsdl.org/release/SDL2-devel-2.0.12-VC.zip), and extract it into the build folder
+* Run the command `cmake -DCMAKE_PREFIX_PATH=SDL2-2.0.12 -DSDL2_LIBRARY_RELEASE=SDL2-2.0.12/lib/x64/*.lib -DSDL2_INCLUDE_DIR=SDL2-2.0.12/include ..` to create project files
```

Comment:
I recommend doing this instead: (Note,  this is bash syntax. Not sure on powershell)

SDLFOLDER=SDL2-2.0.12
cmake -DCMAKE_PREFIX_PATH=$SDLFOLDER -DSDL2_LIBRARY_RELEASE=$SDLFOLDER/lib/x64/*.lib -DSDL2_INCLUDE_DIR=$SDLFOLDER/include ..-DSDL2_INCLUDE_DIR=$SDLFOLDER/include ..